graql/executor/property/RelationExecutor.java

```diff
@@ -170,23 +164,27 @@ public VarProperty property() {
         }
 
         @Override
-        public Set<Variable> requiredVars() {
-            return allVars();
+        public TiebreakDeletionOrdering ordering(WriteExecutor executor) {
+            return TiebreakDeletionOrdering.ROLE_PLAYER;
+        }
```

Comment:
Here as well...

Reply:
yeah only in 1 case do we actually need the WriteExecutor - with `isa`. We need to delete `Relation` instances before other types (and for now implicit attr relations are also deletable using `isa @has...`  for a short while

Reply:
We can change it to not take in a `writeexecutor`?